webapp/locales/de.json

```
@@ -363,7 +363,11 @@
         "label": "Älteste zuerst"
       }
     },
-    "order-by": "Sortieren nach ..."
+    "order-by": "Sortieren nach ...",
+    "save": {
+      "error": "Themen konnten nicht gespeichert werden",
```

```suggestion
      "error": "Themen konnten nicht gespeichert werden!",
```

We put an exclamation mark behind error, I guess.